lib/api/src/grpc/qdrant.rs

```diff
@@ -333,7 +333,7 @@ pub mod collections_client {
     where
         T: tonic::client::GrpcService<tonic::body::BoxBody>,
         T::Error: Into<StdError>,
-        T::ResponseBody: Default + Body<Data = Bytes> + Send + 'static,
+        T::ResponseBody: Body<Data = Bytes> + Send + 'static,
```

Comment:
the diff in generated code must come from https://github.com/qdrant/qdrant/pull/520